docs/controls/page.md

```diff
@@ -823,31 +825,47 @@ page.set_clipboard("This value comes from Flet app")
 </TabItem>
 </Tabs>
 
-### `show_banner(banner: Banner)`
+### `open_banner(banner: Optional[Banner])`
 
 Displays the banner at the top of the page.
 
-### `show_bottom_sheet(bottom_sheet: BottomSheet)`
+:::note
+You can use all open_...() methods without argument if you defined this property. If you didn't defined this property method will raise `Exception`.
+:::
+
+Example:
+```python
+page.drawer = ft.NavigationDrawer(
+    controls=[
+        ft.NavigationDrawerDestination(
+            icon=ft.icons.ADD_TO_HOME_SCREEN_SHARP, label="Item 1"
+        ),
+        ft.NavigationDrawerDestination(icon=ft.icons.ADD_COMMENT, label="Item 2")
+    ]
+)
+page.open_drawer()
+```
+:::
+
+### `open_bottom_sheet(bottom_sheet: Optional[BottomSheet])`
 
 Displays bottom sheet at the bottom of the page.
 
-### `show_dialog(dialog: AlertDialog)`
+### `open_dialog(dialog: Optional[AlertDialog])`
 
 Displays dialog.
 
-### `show_drawer(drawer: NavigationDialog)`
+### `open_drawer(drawer: Optional[NavigationDialog])`
 
 Displays [`drawer`](#drawer).
 
-### `show_end_drawer(drawer: NavigationDialog)`
+### `open_end_drawer(drawer: Optional[NavigationDialog])`
 
 Displays [`end_drawer`](#end_drawer).
 
-### `show_snack_bar(snack_bar: SnackBar)`
+### `open_snack_bar(snack_bar: Optional[SnackBar])`
```

Comment:
As you already know, these `open_*` methods are now deprecated.
To label deprecation in the documentation, [this](https://github.com/flet-dev/website/blob/9f756ce21edfc0792796ba5cd2cc61edaa3722fc/docs/controls/circleavatar.md?plain=1#L99-L103) is how we do it at the moment. (strikethrough the name, then add little note below concerning the deprecation and in case of a rename, we also add what is now to be used).

Before deprecation, you have to bring these methods back to their previous state first (because the `show_*` methods are those being deprecated, not the `open_*`).

If you can't do it or don't understand what I mean, don't worry, just let me know and I will do it. 😉


Please can you add `open()` and `close()` methods in here?

Reply:
> As you already know, these `open_*` methods are now deprecated in favor of single `open()` and `close()`. To label deprecation in the documentation, [this](https://github.com/flet-dev/website/blob/9f756ce21edfc0792796ba5cd2cc61edaa3722fc/docs/controls/circleavatar.md?plain=1#L99-L103) is how we do it at the moment. (strikethrough the name, then add little note below concerning the deprecation and in case of a rename, we also add what is now to be used).
> 
> Before deprecation, you have to bring these methods back to their previous state first (because the `show_*` methods are those being deprecated, not the `open_*`).
> 
> If you can't do it or don't understand what I mean, don't worry, just let me know and I will do it. 😉

Please, can you merge my PR and fix this yourself?